src/pages/home/library/Mint.tsx

```diff
@@ -0,0 +1,451 @@
+              These details cannot be changed after minting.
+            </Typography>
+          </Alert>
+        </Stack>
+      ) }
+
+      { isFirstStep ? (
+        <>
+          <Box
+            sx={ {
+              backgroundColor: theme.colors.grey600,
+              mt: 4,
+              p: 3,
+            } }
+          >
+            <Stack
+              spacing={ 1 }
+              sx={ {
+                alignItems: "center",
+                display: "flex",
+                flexDirection: [null, null, "row"],
+                justifyContent: "space-between",
+              } }
+            >
+              <Stack spacing={ 1 } sx={ { maxWidth: "500px" } }>
+                <Typography>Mint Song</Typography>
+                <Typography fontSize={ 12 } variant="subtitle1">
+                  Minting a song will create an NFT that reflects ownership,
+                  making streaming royalties purchasable. Once a song is minted,
+                  it cannot be deleted.
+                </Typography>
+              </Stack>
+
+              <Switch
+                checked={ isMintSwitchOn }
+                onChange={ () => {
+                  setIsMintSwitchOn(!isMintSwitchOn);
+                } }
+                sx={ { mt: [2, 2, 0] } }
+              />
+            </Stack>
+
+            { isMintSwitchOn && (
+              <>
+                <HorizontalLine sx={ { my: 4 } } />
+
+                { futureOwners.length ? (
+                  <>
+                    <Stack flexDirection="row" justifyContent="space-between">
+                      <Typography color="grey100" variant="h5">
+                        MASTER OWNERS
+                      </Typography>
+                      <Typography color="grey100" variant="h5">
+                        SHARES
+                      </Typography>
+                    </Stack>
+                    { futureOwners.map((owner: FutureOwner) => (
+                      <Stack
+                        key={ owner.email }
+                        sx={ {
+                          flexDirection: "row",
+                          justifyContent: "space-between",
+                          mt: 1.5,
+                        } }
+                      >
+                        <Stack>
+                          <Typography>{ `${owner.firstName} ${owner.lastName}` }</Typography>
+                          <Typography variant="subtitle1">
+                            { owner.email }
+                          </Typography>
+                        </Stack>
+                        <Stack flexDirection="row" alignItems="center">
+                          <TextInput
+                            max={ 100 }
+                            min={ 0 }
+                            onChange={ (event) =>
+                              handlePercentageChange(event, owner.email)
+                            }
+                            placeholder="0 %"
```

Comment:
Is there a way to keep the percent symbol visible as the numbers change? Otherwise the `AddOwner.tsx` uses the following within their `TextInputField`:
```
endAdornment={
                <InputAdornment
                  position="start"
                  sx={ {
                    color: theme.colors.white,
                    mr: 1,
                  } }
                >
                  <Typography>%</Typography>
                </InputAdornment>
              }
```

Reply:
oh that's an interesting approach, i will check it out and make it consistent.